packages/block-library/src/block/test/edit.native.js

```diff
@@ -72,25 +138,19 @@ describe( 'Reusable block', () => {
 			getByA11yLabel( /Reusable block Block\. Row 1/ )
 		);
 
-		const reusableBlockRequest = fetchRequest.mock.calls.find( ( call ) =>
-			call[ 0 ].path.startsWith( endpoint )
-		);
-
 		const blockDeleted = await waitFor( () =>
 			within( reusableBlock ).getByText(
 				'Block has been deleted or is unavailable.'
 			)
 		);
 
-		expect( reusableBlockRequest ).toBeDefined();
 		expect( reusableBlock ).toBeDefined();
 		expect( blockDeleted ).toBeDefined();
-		expect( getEditorHtml() ).toBe( initialHtml );
```

Comment:
We're not making modifications to the content so this check is not really necessary.